argilla/docs/how_to_guides/distribution.md

```diff
@@ -0,0 +1,73 @@
+---
+description: In this section, we will provide a step-by-step guide to show how to distribute the annotation task among team members.
+---
+
+# Distribute the annotation among the team
```

Comment:
Unsure about the title here too

Reply:
Do you think "Distribute the annotation task among the team" makes more sense to you?